compiler/test/compilable/ddoc10236.d

```diff
@@ -1,5 +1,5 @@
 // PERMUTE_ARGS:
-// REQUIRED_ARGS: -D -Dd${RESULTS_DIR}/compilable -wi -o-
+// REQUIRED_ARGS: -verrors=simple -D -Dd${RESULTS_DIR}/compilable -wi -o-
```

Comment:
`-verrors=simple` is set by the test runner now, correct? If so you need not specify it here. Ditto throughout

Reply:
only for FAIL_COMPILE files, not for COMPILE or RUN files (there's some bug, idk what exactly- but log can give you some insights, maybe? https://github.com/dlang/dmd/actions/runs/12408528891/job/34640372421?pr=20576, that's why I manually updated the COMPILE test files)

Reply:
Oh. Those are deprecations that we run with `-dw` (deprecations as warnings, the default) instead of `-de` (deprecations as errors).

Reply:
Ignore this then